samples/extensions/descriptor_buffer_basic/descriptor_buffer_basic.cpp

```diff
@@ -0,0 +1,471 @@
+/* Copyright (c) 2023, Sascha Willems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Using descriptor buffers replacing descriptor sets with VK_EXT_descriptor_buffer
+ * This render multiple cubes passing uniform buffers and combined image samples to the GPU using descriptor buffers instead of descriptor sets
+ * This allows for a more bindless design
+ */
+
+#include "descriptor_buffer_basic.h"
+
+#include "core/buffer.h"
+#include "scene_graph/components/sub_mesh.h"
+
+DescriptorBufferBasic::DescriptorBufferBasic()
```

Comment:
descriptor_set_layout_buffer, descriptor_set_layout_image, pipeline, pipeline_layout are not initialized.